src/connector.js

```diff
@@ -0,0 +1,80 @@
+const winston = require('winston');
+const { ConnectionStringParser } = require('connection-string-parser')
+const { ReplSet, Server, MongoClient } = require('mongodb');
+
+const connectionStringParser = new ConnectionStringParser({
+  scheme: "mongodb",
+  hosts: []
+});
+
+const connector = {
+  client: null,
+
+  connected: false,
+
+  connectionString: '',
+
+  connect: async (connectionString) => {
```

Comment:
have an object as a parameter - 
{ 
    connectionString, databaseName
}

Reply:
Doesn't connection string contain the database name also?